test/plugins/test_substitute.py

```diff
@@ -0,0 +1,116 @@
+# This file is part of beets.
+# Copyright 2024, Nicholas Boyd Isacsson.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Test the substitute plugin regex functionality."""
+
+from beets.test.helper import PluginTestCase, capture_log
+from beetsplug.substitute import Substitute
+
+PLUGIN_NAME = "substitute"
```

Comment:
Is this required?

Reply:
Apparently not, thank you! I saw it in another test (advanced_rewrite) and assumed it was required.